src/prefect/agent/kubernetes/agent.py

```diff
@@ -127,21 +132,12 @@ def __init__(
         )
 
         from prefect.utilities.kubernetes import get_kubernetes_client
-        from kubernetes import client, config
 
-        try:
-            self.logger.debug("Loading incluster configuration")
-            config.load_incluster_config()
-        except config.config_exception.ConfigException as exc:
-            self.logger.warning(
-                "{} Using out of cluster configuration option.".format(exc)
-            )
-            self.logger.debug("Loading out of cluster configuration")
-            config.load_kube_config()
```

Comment:
This is run in `get_kubernetes_client` and there is no reason to run it here as well